workflow/util.py

```diff
@@ -0,0 +1,17 @@
+from typing import Any, Dict
+
+
+def result(success: Any, data: Dict[str, Any] = None, message: str = None, details: str = None):
+    """
+        Default result dict structure used by all components of the module
+    """
+    result_dict = {'success': success}
+
+    if data:
+        result_dict['data'] = data
+    if message:
+        result_dict['message'] = message
+    if details:
+        result_dict['details'] = details
+
+    return result_dict
```

Comment:
We could implement it as a data class, but i made the 'data' field to be any dict so it would not be fully typed, so it kind of defeats the point of typing the messages.